packages/cardano-services/src/Metadata/mappers.ts

```diff
@@ -1,10 +1,21 @@
-import { Cardano, ProviderUtil } from '@cardano-sdk/core';
+import { CML, Cardano, cmlToCore } from '@cardano-sdk/core';
 import { TxMetadataModel } from './types';
+import { usingAutoFree } from '@cardano-sdk/util';
 
-export const mapTxMetadata = (metadataModel: Pick<TxMetadataModel, 'json_value' | 'key'>[]): Cardano.TxMetadata =>
+export const mapTxMetadata = (metadataModel: Pick<TxMetadataModel, 'bytes' | 'key'>[]): Cardano.TxMetadata =>
   metadataModel.reduce((map, metadatum) => {
-    const { key, json_value } = metadatum;
-    if (!json_value || !key) return map;
-    map.set(BigInt(key), ProviderUtil.jsonToMetadatum(json_value));
+    const { bytes, key } = metadatum;
+
+    if (bytes && key) {
+      const biKey = BigInt(key);
+      const datumMap = usingAutoFree((_) => cmlToCore.txMetadatum(CML.TransactionMetadatum.from_bytes(bytes)));
```

Comment:
- `datum` refers to an output field in Cardano, so best not to use it here
- Object that's being parsed here appears to have label, so we should be mapping the correct core object here (`Metadata` vs `Metadatum`)

```suggestion
      const metadata = usingAutoFree((_) => cmlToCore.txMetadata(CML.TransactionMetadatum.from_bytes(bytes)));
```

Reply:
7c51e61d6 should solve this